app/models/messages/EconomicOperator.scala

```diff
@@ -18,9 +18,7 @@ package models.messages
 
 import play.api.libs.json.{Json, OFormat}
 
-case class EconomicOperator(identificationNumber: String) {
-  override def toString: String = identificationNumber
-}
+case class EconomicOperator(identificationNumber: String)
```

Comment:
Are we sure the toString method isn't being used somewhere? Sometimes hard to tell. Don't think it's doing much harm

Reply:
Added it back just in case